pulp_smash/utils.py

```diff
@@ -9,7 +9,7 @@
 from urllib.parse import urlparse
 
 import requests
-
```

Comment:
Please leave this blank line here. From [PEP 8](https://www.python.org/dev/peps/pep-0008/):

> Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.
>
> Imports should be grouped in the following order:
>
> * Standard library imports.
> * Related third party imports.
> * Local application/library specific imports.
>
> You should put a blank line between each group of imports.